exporter/parquetexporter/README.md

```diff
@@ -0,0 +1,28 @@
+# Parquet File Exporter
+
+Sends pipeline data to Parquet files.
+
+Supported pipeline types: logs, metrics, traces
+
+> :construction: This receiver is in beta and configuration fields are subject to change.
+
+## Configuration
+
+The following configuration options are required:
+
+- `path` (no default): Export Parquet file path.
+
+The following configuration options can also be configured:
+
+TODO
+
+Example:
+
+```yaml
+exporters:
+  parquet:
+    path: /var/output/log.parquet
```

Comment:
Just want to check whether the intention is to be able to do something like `s3://bucket/path`, `gcs://bucket/path` here too to get cloud storage in the same component? Notably, hopefully the implementation uses a filesystem implementation that supports cloud backends so it's easy to do this, if it just uses normal filesystem then it would require a significant work to then redo it to support cloud

Reply:
Oops sorry for commenting on such an old PR, got confused since the issue had a recent update

Reply:
I think this is to write to parquet, and the transport layer in scope is the filesystem until someone comes along and makes it better